lib/stanford-mods/geo_spatial.rb

```diff
@@ -1,27 +1,48 @@
 # encoding: UTF-8
-# require 'logger'
 require 'mods'
 
 module Stanford
   module Mods
     # NON-SearchWorks specific wranglings of MODS cartographics metadata
     class Record < ::Mods::Record
+      GMLNS = 'http://www.opengis.net/gml/3.2/'.freeze
+
+      # @return [Array{String}] subject cartographic coordinates values
       def coordinates
         Array(@mods_ng_xml.subject.cartographics.coordinates).map(&:text)
       end
 
+      # @return [Array{String}] values suitable for solr SRPT fields, like "ENVELOPE(-16.0, 28.0, 13.0, -15.0)"
+      # @note example xml
+      #  <gml:lowerCorner>-122.191292 37.4063388</gml:lowerCorner>
+      #  <gml:upperCorner>-122.149475 37.4435369</gml:upperCorner>
+      def geo_extensions_as_envelope
+        @mods_ng_xml.extension
+                    .xpath('//rdf:RDF/rdf:Description') # separate to avoid having to specify rdf namespace
```

Comment:
We should not depend on any particular prefix. It could change upstream. So please add the namespace that you want `rdf` to represent.